components/scrollable-container/demo/demo.html

```diff
@@ -1,59 +1,83 @@
+                Nam at justo enim. Nam dictum facilisis mattis. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Fusce eget blandit ex, nec elementum erat. Vivamus purus purus, bibendum quis hendrerit sed, vehicula vitae arcu. Nam enim ligula, rutrum vitae imperdiet vitae, tristique id urna. Aenean rutrum sed nunc vel ultricies. Suspendisse iaculis, dolor vel blandit blandit, lacus tellus sodales nulla, id aliquam est nisl eu ligula. Nulla fermentum neque quis metus tristique scelerisque. Nulla aliquam vel libero sit amet mollis. Nulla ut consequat nisl. Proin eu dignissim nisi.
+            </component-slot>
+        </gameface-scrollable-container>
 
+        <div>
+            <gameface-scrollable-container class="scrollable-container-component fixed-width" automatic>
+                <component-slot data-name="scrollable-content">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In eu urna tempus, ultricies lacus
+                    fermentum,
+                    posuere arcu. Ut eget elit magna. Interdum et malesuada fames ac ante ipsum primis in faucibus.
+                    Suspendisse
+                    <div class="dynamic-content">
```

Comment:
Has some dynamic contend.
Also tried with .textContent without an extra hidden element and it works, but it breaks more often in cohtml due to the mutatioObserver callback not being called sometimes issue - so I left it with the extra element.